tools/Crank/run-benchmarks.ps1

```diff
@@ -98,6 +98,14 @@ if ($WriteResultsToDatabase) {
     $crankArgs += '--table', 'FunctionsPerf'
 }
 
-& $InvokeCrankCommand $crankArgs
+& $InvokeCrankCommand $crankArgs 2>&1 | Tee-Object -Variable crankOutput
+
+$badResponses = $crankOutput | Where-Object { $_ -match '\bBad responses\b\s*\|\s*(\S*)\s' } | ForEach-Object { $Matches[1] }
+if ($null -eq $badResponses) {
+    Write-Warning "Could not detect the number of bad responses. The performance results may be unreliable."
```

Comment:
This will only work for bombardier though? If so, might make sense to skip this warning, so we just have a bombardier specific check below.

Reply:
Though I guess if we switch load generators, this warning will remind us to update the check...

Reply:
Yes, this is the idea.